CredentialProvider.Microsoft/Program.cs

```diff
@@ -87,8 +87,9 @@ public static async Task<int> Main(string[] args)
                             EnvUtil.SessionTimeEnvVar,
                             EnvUtil.TokenTypeEnvVar,
                             EnvUtil.BuildTaskUriPrefixes,
-                            EnvUtil.BuildTaskAccessToken,
                             EnvUtil.BuildTaskExternalEndpoints,
+                            EnvUtil.EndpointCredentials,
```

Comment:
Is Resources.EnvironmentVariableHelp updated to match? don't see that unless I'm blind here

Reply:
updated